src/test/java/org/candlepin/subscriptions/db/HostRepositoryTest.java

```diff
@@ -509,7 +509,7 @@ void testCanSortByDisplayName() {
                 0,
                 10,
                 Sort.Direction.ASC,
-                HostsResource.HOST_SORT_PARAM_MAPPING.get(HostReportSort.DISPLAY_NAME)));
+                InstancesResource.HOST_SORT_PARAM_MAPPING.get(HostReportSort.DISPLAY_NAME)));
```

Comment:
These should be updated to INSTANCE_SORT_PARAM mappings (if there are any that don't map we should remove the resulting dead code).